p2p-specs/p2p-interface.md

```diff
@@ -112,13 +113,15 @@ Bundlers MUST locally store the following `MetaData`:
 (
   seq_number: uint64
   mempool_nets: Bitvector[MEMPOOL_SUBNET_COUNT]
+  supported_mempools: List[Bytes32, MAX_SUPPORTED_MEMPOOLS]
```

Comment:
why do you need to define the `supported_mempools ` member, and why there's a limit ? 
the bitmap of supported mempools is 1024-bit wide, and indeed is optimized if there are less, but nothing prevent a bundler to support more.

Reply:
In a previous discussion we dropped the bitmap of supported mempools from the ENR, this is the only location where users communicate to peers the list of mempools they support (so peers can disconnect when whey don't share a common pool they need peers on).

`MAX_SUPPORTED_MEMPOOLS` is there because every message size in this protocol needs to be constrained somehow to prevent spamers.